routes/authRoutes.js

```diff
@@ -0,0 +1,67 @@
+const express = require('express');
+const Router = express.Router();
+const Users = require('../db/models/users.js')
+
+Router.post('/auth/register', (req, res) => {
+    const {user_id, email, password} = req.body;
+        Users
+            .forge({user_id, email, password})
+            .save()
+            .then( result => {
+                if (result){
+                    //save user data into DB
+                    console.log('WELCOME')
+                }
```

Comment:
Look out for indentation consistency, it helps with readability and preventing syntax errors.